graylog2-web-interface/src/util/AppConfig.js

```diff
@@ -30,6 +30,15 @@ const AppConfig = {
     return this.appConfig().rootTimeZone;
   },
 
+  isCloud() {
+    if (typeof (IS_CLOUD) !== 'undefined') {
```

Comment:
You can remove the parenthesis after `typeof`
```suggestion
    if (typeof IS_CLOUD !== 'undefined') {
```